src/Core/Models/Response/ConfigResponse.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Bit.Core.Models.Response
+{
+    public class ConfigResponse
+    {
+        public string Version { get; set; }
+        public string GitHash { get; set; }
+        public ServerConfigResponse Server { get; set; }
+        public EnvironmentConfigResponse Environment { get; set; }
+        public IDictionary<string, object> FeatureStates { get; set; }
```

Comment:
Can we have a better default type than `object`?

Reply:
#2454 will be merged into this branch in order to be tested as a whole before it goes into master.